app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class UserMailer < ApplicationMailer
+  default from: 'czarrrna@gmail.com'
+
+  def welcome_email(user)
```

Comment:
It's better to pass just user#id as mailer tend to be run by some background job manager like delayed_job or sidekiq. In that case data is serialized and if pass whole object, this is object is serialized and that often leads to unexpected outcomes. So pass user#id and find that user in mailer.